segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentStoreMetrics.java

```diff
@@ -148,20 +150,20 @@ private void report() {
          */
         private final OpStatsLogger processOperationsLatency;
         private final OpStatsLogger processOperationsBatchSize;
-        private final String operationLogSize;
+        private final String[] containerTag;
```

Comment:
Really minor comment, this should be `containerTags` plural as it is an array of tags.

Reply:
Yes, one tag is a "pair" of strings. Here containerTag and hostTag returns "one" pair of strings as "a" tag, while all others, such as streamTags, returns multiple pairs of strings as multiple tag"s".

So precisely the method might be called "containerTagPair", "streamTagPairs". The method name determines plural or single from "pair" perspective instead of String perspective, but I agree it's not straightforward sometimes to readers.